.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -213,7 +200,6 @@ fun applyDefaults(model: CIBuildModel, buildType: BaseGradleBuildType, gradleTas
 
     buildType.checkCleanM2Step(os)
     buildType.verifyTestFilesCleanupStep(daemon)
-    buildType.tagBuildStep(model, daemon)
```

Comment:
Is there any chance we can find a flaky build without using the `re-runner`? If there is, we would need to keep this step. If it doesn't make any sense anymore, by all means let's remove the step.

Reply:
I believe there won't be any.

https://github.com/gradle/ci-health/blob/d6c9c228fdedfe037a72b415fc0faae82503cd51/gradle-build-tag-plugin/src/main/groovy/org/gradle/ci/tagging/TeamCityFailedBuildTagger.groovy#L198

```
        if (!teamCityHttpClient.getBuildSteps(build).contains("GRADLE_RERUNNER")) {
            return false
        }
```

If there's no `GRADLE_RERUNNER` step, a build will never be judged as flaky.